.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        versions:
+          - python: "2.6"
+          - python: "2.7"
+          - python: "3.3"
+          - python: "3.4"
+          - python: "3.5"
+          - python: "3.6"
+          - python: "3.7"
+          - python: "3.8"
+          - python: "3.9"
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - if: ${{ matrix.versions.python != '2.6' }}
+        run: |
+          docker build \
+          . \
+          --build-arg PYTHON_VERSION=${{ matrix.versions.python }} \
+          -t jmadler/python-future-builder:${{ matrix.versions.python }}
+      - if: ${{ matrix.versions.python == '2.6' }}
+        run: |
+          docker build \
+          . \
+          -f 2.6.Dockerfile \
+          -t jmadler/python-future-builder:${{ matrix.versions.python }}
```

Comment:
If you want to push the images to dockerhub, you might want to use [this GitHub action](https://github.com/docker/build-push-action), but it'll require a maintainer to setup the credentials.

Reply:
SGTM.  @edschofield you may also want to retag as under the PythonCharmers org instead of under my personal profile.